bumble/gatt_server.py

```diff
@@ -79,6 +80,45 @@ def get_attribute(self, handle):
                 return attribute
         return None
 
+    def get_service_attribute(self, service_uuid: UUID) -> Service:
+        for attribute in self.attributes:
```

Comment:
more concise with an iterator: `return next((attribute in self.attribute if attribute.type == GATT_PRIMARY_SERVICE_ATTRIBUTE_TYPE and attribute.uuid == service_uuid), None)`

Reply:
updated, let me know what you think